spec/factories/contacts.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: UTF-8
 FactoryGirl.define do
   factory :contact do |c|
-    name
+    name "Spindelmän"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.